test.js

```diff
@@ -101,8 +101,51 @@ test('don\'t delete files, but return them - sync', t => {
 	});
 	exists(t, ['1.tmp', '2.tmp', '3.tmp', '4.tmp', '.dot.tmp']);
 	t.deepEqual(deletedFiles, [
-		path.join(t.context.tmp, '2.tmp'),
+		path.join(t.context.tmp, '4.tmp'),
 		path.join(t.context.tmp, '3.tmp'),
-		path.join(t.context.tmp, '4.tmp')
+		path.join(t.context.tmp, '2.tmp')
 	]);
 });
+
+// Currently this is only testable locally on macOS.
```

Comment:
I don't understand this comment. Travis is passing with this test, and Travis runs on Linux.

Reply:
The test will pass on travis without the sorting change but will fail on a computer running macOS. Adding osx to travis's testing matrix does not help.

Does that make sense?

Reply:
Yup